src/blocks/blocks/progress-bar/edit.js

```diff
@@ -91,7 +94,17 @@ const ProgressBar = ({
 		};
 	}, [ attributes.percentage, attributes.duration ]);
 
-	const fontRatio = 0.65;
+	const styles = {
+		'--titleColor': attributes.titleColor,
+		'--percentageColor': attributes.percentageColor,
+		'--percentageColorOuter': attributes.percentageColor,
+		'--percentageColorTooltip': attributes.percentageColor,
+		'--percentageColorAppend': attributes.percentageColor,
+		'--backgroundColor': attributes.backgroundColor,
+		'--borderRadius': attributes.borderRadius !== undefined && ( attributes.borderRadius + 'px' ),
+		'--height': attributes.height !== undefined && ( attributes.height + 'px' ),
+		'--barBackground': attributes.percentageColor
+	};
```

Comment:
@HardeepAsrani, I think this is the most reliable way of handling the CSS var in js. Now, we need to find a 
 reliable solution for handling things like `:hover`.